src/com/cje/RequestPage.java

```diff
@@ -1,17 +1,633 @@
+    private JFormattedTextField headTxt;
+    private JFormattedTextField bmiTxt;
+    private JButton viewBtn;
+    private JFormattedTextField birthWeightTxt;
+    private JFormattedTextField birthHeightTxt;
+    private JLabel ageTxt;
+    private JLabel baseDateTxt;
+    private JFormattedTextField birthDateTxt;
+    private JButton resetBtn;
+    private JButton exitBtn;
+    private JPanel resultPanel;
+    private JLabel resultTxt;
+    private JCheckBox calculateByDayCkBox;
+    private JButton changeDateBtn;
+    private JLabel ageLabel;
+
+    private DataManager dm;
+    private ResultView rv;
+    private Params.Gender gender = Params.Gender.NONE;
+    private int birthYear;
+    private int birthMonth;
+    private int birthDay;
+    private LocalDate baseDate;
+
+    private float birthWeight;
+    private float birthHeight;
+    private float height;
+    private float weight;
+    private float head;
+    private float bmi;
+
+    private int ageByDay;
+
+    boolean isDataLoaded;
+    boolean isCalculatedByDay;
+
+    // Initialize member variables
+    private void initParam() {
+        birthYear = -1;
+        birthMonth = -1;
+        birthDay = -1;
+        ageByDay = -1;
+
+        birthWeight = -1;
+        birthHeight = -1;
+        height = -1;
+        weight = -1;
+        head = -1;
+        bmi = -1;
+
+        setBaseDate(LocalDate.now());
+    }
+
+    private void init() {
+        initParam();
+
+        dm = new DataManager();
+        rv = new ResultView();
+
+        // Load Data
+        isDataLoaded = false;
+        isCalculatedByDay = true;
+
+        isDataLoaded = dm.loadData();
+        if(!isDataLoaded) {
+            Utils.showMessage("Data loading failed!");
+            Utils.LOGGER.log(Level.SEVERE, "File reading failed.");
+        }
+    }
+
+    public RequestPage() {
+        super("Growth Indicator");
+
+
+        setContentPane(mainPanel);
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        pack();
+        setLocationRelativeTo(null);
+
+        // initialize variables and the objects
+        init();
+
+        viewBtn.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if(!isDataLoaded) {
+                    Utils.showMessage("파일이 정상적으로 로딩되지 않았습니다.");
+                    return;
+                }
+
+                // Check Gender
+                if(maleRdBtn.isSelected()) gender = Params.Gender.MALE;
+                else if(femaleRdBtn.isSelected()) gender = Params.Gender.FEMALE;
+                else {
+                    Utils.showMessage("Please select gender.");
+                    return;
+                }
+
+                // Check Weight at Birth
+                if(birthWeightTxt.getText().length()!=0) {
+                    if(!Utils.isActualNumber(birthWeightTxt.getText())) {
+                        Utils.showMessage("Please input number correctly");
```

Comment:
Please use specific message.
In this case, "Please use valid birth weight." would be better.